Aptoide/aptoide-client-v8
# [ASV-1350] Hide TC and PP from cobrand flavor.

**What does this PR do?**

   This PR aims to remove the TC and PP from the cobrand flavor. This was achieved by handling the show/hide on LoginSignupCredentials and PaymentLogin with two different presenters, one for each flavor. These presenters extend an abstract "base" presenter. For the settings part, simply used the manager created earlier to show/hide the preferences entries.

**Database changed?**

   No

**Where should the reviewer start?**

- [ ] LoginSignupCredentialsPresenter.java
- [ ] LoginSignupCredentialsFlavorPresenter.java
- [ ] LoginSignupCredentialsPresenter.java
- [ ] LoginSignupCredentialsPresenter.java
- [ ] PaymentLoginFragment.java
- [ ] PaymentLoginPresenter.java
- [ ] PaymentLoginFlavorPresenter.java
- [ ] SettingsFragment.java

**How should this be manually tested?**

  Check both cobrand and vanilla versions, and see if the TC and PP are being shown/hidden in the correct flavors. Logins, settings.

**What are the relevant tickets?**

  https://aptoide.atlassian.net/browse/ASV-1350



**Code Review Checklist**
